src/layout/dashboard/header/DashboardHeader.js

```diff
@@ -113,9 +142,18 @@ export default function DashboardHeader() {
   const pathname = usePathname();
   const { dashboardType } = useAppSelector((store) => store.app);
   const isMobile = useMediaQuery(theme.breakpoints.down('md'));
+  const dispatch = useAppDispatch();
   const [isDrawerMobileOpen, setIsDrawerMobileOpen] = useState(false);
 
-  const dispatch = useAppDispatch();
+  const onClickToggleDrawerMobile = () => (event) => {
+    if (
+      event.type === 'keydown' &&
+      (event.key === 'Tab' || event.key === 'Shift')
+    ) {
+      return;
+    }
+    setIsDrawerMobileOpen(!isDrawerMobileOpen);
```

Comment:
use previous state to handle, not current state state to handle, not current state

[ref](https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state)

Reply:
dạ dể em thử làm lại

Reply:
dạ em thử rồi ạ anh rv giúp em

Reply:
có gì khác đâu, đổi thành 
`setIsDrawerMobileOpen((prev) => !prev);`

Reply:
bữa đổi nó ko ăn á anh

Reply:
nó bị lỗi bữa em gửi á